### ISSUE TITLE
Missing import  in  site-packages/awx/main/tasks.py

### ISSUE THREAD

**Issue by TryHardDoMore** [MrMEEE/awx-build#76]
* Python version: 3.6.3
* AWX version: 3.0.1
* Repo used: https://copr.fedorainfracloud.org/coprs/mrmeee/ansible-awx/repo/epel-7/mrmeee-ansible-awx-epel-7.repo

* Details: Stucked in "Not Started" scheduled job:
* Logs: 
```
Feb 10 23:15:13 : 2019-02-11 04:15:13,626 ERROR    awx.main.tasks Error spawning scheduled job.
Feb 10 23:15:13 : Traceback (most recent call last):
Feb 10 23:15:13 : File "/opt/rh/rh-python36/root/usr/lib/python3.6/site-packages/awx/main/tasks.py", line 471, in awx_periodic_scheduler
Feb 10 23:15:13 : logger.info(six.text_type('Spawned {} from schedule {}-{}.').format(
Feb 10 23:15:13 : NameError: name 'six' is not defined
```
* Solution: adding missing import helps.

**Comment by MrMEEE** [comment]
Hi

Thanks for reporting.. In which log do you see this?

When you write "adding missing import".. do you mean that you install six or that you do an import in the python file??

**Comment by MrMEEE** [comment]
This seems to be an upstream bug, which is already fixed.. 

https://github.com/ansible/awx/issues/3200

Which version of 3.0.1 are you running:

rpm -aq ansible-awx

**Comment by TryHardDoMore** [comment]
I mean just adding import to file itself :)
* Version: ansible-awx-3.0.0.128-1.el7.x86_64

**Comment by MrMEEE** [comment]
ahh.. so not 3.0.1... 

This was fixed around 3.0.1.2-1.el7.. 

Please try to update..

**Comment by TryHardDoMore** [comment]
ow, ma mistake. Thx :)
